android/src/main/java/com/reactnativechangeicon/ChangeIconModule.kt

```diff
@@ -50,11 +50,14 @@ class ChangeIconModule(reactContext: ReactApplicationContext, private val packag
           if (activity == null) {
             return
           }
-          activity.packageManager.setComponentEnabledSetting(
-            ComponentName(packageName, classToKill),
-            PackageManager.COMPONENT_ENABLED_STATE_DISABLED,
-            PackageManager.DONT_KILL_APP
-          )
+          for (c in classesToKill) {
```

Comment:
try using an iterator function instead of a for loop

Reply:
Also updated.